test/orfs/mock-array/BUILD

```diff
@@ -200,6 +200,7 @@ orfs_flow(
         # some width in multiply reduction going on here to speed up
         # builds, so we want to keep this exact module.
         "SYNTH_KEEP_MODULES": "Multiplier",
+        "SYNTH_HIERARCHICAL": "1",
```

Comment:
remove "SYNTH_HIERARCHICAL": "1". It is superfluous with "SYNTH_KEEP_MODULES": "Multiplier".

SYNTH_HIERARCHICAL drives a policy to select which modules to keep, "SYNTH_KEEP_MODULES": "Multiplier" is an explicit list.